gulp-tasks/utils/upload-bundle-to-cdn.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const glob = require('glob');
+
+const spawnPromiseWrapper = require('./spawn-promise-wrapper');
+
+const PROJECT_ID = 'workbox-bab1f';
+const BUCKET_NAME = 'workbox-cdn';
+
+module.exports = (tagName, directoryToUpload) => {
+  return spawnPromiseWrapper('gcloud', [
```

Comment:
Since there's an official `node` client library for the Google Cloud stuff, what do you think about switching to that? It also means we'll have everything needed to publish listed as dependencies in `package.json`, rather than needing to install the Google Cloud CLI out of band.

https://cloud.google.com/appengine/docs/flexible/nodejs/using-cloud-storage